src/OT/glyf/glyf.hh

```diff
@@ -98,7 +98,9 @@ struct glyf
       use_short_loca = max_offset < 0x1FFFF;
     }
 
-    glyf_prime->serialize (c->serializer, glyphs.writer (), use_short_loca, c->plan, font);
+    if (!glyf_prime->serialize (c->serializer, glyphs.writer (), use_short_loca, c->plan, font))
```

Comment:
serialization could fail part way through which means some of the glyphs will have allocated memory but not all, so we still need to run the loop to free memory, but will need a way to track which SubsetGlyphs have allocated memory and which are references.

Reply:
yes I missed that. I'll work on it.